R/util.R

```diff
@@ -196,7 +196,12 @@ pasteq <- function(x, sep = ", ") {
 
 capture_log <- function(expr, filename) {
   ## nolint start
-  con <- file(filename, "w")
+  if (file.exists(filename)) {
+    mode = "a"
```

Comment:
why not `<-`? and why is nolint required?

Reply:
you can also just write 

```
mode <- if (file.exists(filename)) "a" else "w"
```

Reply:
> why not `<-`? and why is nolint required?

Just being absent minded. I've got rid of the nolint stuff too.